express-api/src/controllers/lookup/lookupController.ts

```diff
@@ -47,17 +56,82 @@ export const lookupRoles = async (req: Request, res: Response) => {
  * @param {Response}    res Outgoing response
  * @returns {Response}      A 200 status and a list of property classifications.
  */
-export const lookupPropertyClassifications = async (req: Request, res: Response) => {
+export const lookupPropertyClassifications = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
   /**
    * #swagger.tags = ['Lookup']
    * #swagger.description = 'Get all property classification entries.'
    * #swagger.security = [{
             "bearerAuth": []
       }]
    */
+  try {
+    const classifications = await AppDataSource.getRepository(PropertyClassification).find();
+    const filtered = classifications.filter((c) => !c.IsDisabled);
+    const parsed = ClassificationPublicResponseSchema.array().safeParse(filtered);
+    if (parsed.success) {
+      return res.status(200).send(parsed.data);
+    } else {
+      return res.status(400).send('Something went wrong.');
+    }
+  } catch (e) {
+    next(e);
+  }
+};
 
-  // TODO: Replace stub response with controller logic
-  return stubResponse(res);
+export const lookupBuildingPredominateUse = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
+) => {
+  /**
+   * #swagger.tags = ['Lookup']
+   * #swagger.description = 'Get all predomanite uses entries.'
+   * #swagger.security = [{
+            "bearerAuth": []
+      }]
+   */
+  try {
+    const uses = await AppDataSource.getRepository(BuildingPredominateUse).find();
+    const filtered = uses.filter((u) => !u.IsDisabled);
+    const parsed = PredominateUsePublicResponseSchema.array().safeParse(filtered);
+    if (parsed.success) {
+      return res.status(200).send(parsed.data);
+    } else {
+      return res.status(400).send('Something went wrong.');
```

Comment:
It's possible to do this, but it seems like the return type of safeParse is typed incorrectly, since I can't access that property without a compiler error. Only seems to work if you force a type cast. Are you able to make it work cleanly?